platforms/ios/framework/src/TGMapView.h

```diff
@@ -0,0 +1,913 @@
+ */
+@property (readonly, nonatomic) SEL respondToDoubleTapGestureAction;
+
+/**
+ The selector that must be used as an action on the pan gesture recognizer
+ */
+@property (readonly, nonatomic) SEL respondToPanGestureAction;
+
+/**
+ The selector that must be used as an action on the pinch gesture recognizer
+ */
+@property (readonly, nonatomic) SEL respondToPinchGestureAction;
+
+/**
+ The selector that must be used as an action on the rotation gesture recognizer
+ */
+@property (readonly, nonatomic) SEL respondToRotationGestureAction;
+
+/**
+ The selector that must be used as an action on the shove gesture recognizer
+ */
+@property (readonly, nonatomic) SEL respondToShoveGestureAction;
+
+/**
+ The selector that must be used as an action on the long press gesture recognizer
+ */
+@property (readonly, nonatomic) SEL respondToLongPressGestureAction;
+
+#pragma mark Gesture Recognizers
+
+/**
+ The following recognizers must be set after viewDidLoad in the view lifecycle
+ in order to override default. Only one instance of the recognizer type will be
+ used on the view at once. Adding any recognizer removes the default or
+ previously added recognizer on the map view.
+ */
+
+/**
+ Replaces the tap gesture recognizer used by the map view and adds it to the
+ UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UITapGestureRecognizer* tapGestureRecognizer;
+
+/**
+ Replaces the double tap gesture recognizer used by the map view and adds it to
+ the UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UITapGestureRecognizer* doubleTapGestureRecognizer;
+
+/**
+ Replaces the pan gesture recognizer used by the map view and adds it to the
+ UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UIPanGestureRecognizer* panGestureRecognizer;
+
+/**
+ Replaces the pinch gesture recognizer used by the map view and adds it to the
+ UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UIPinchGestureRecognizer* pinchGestureRecognizer;
+
+/**
+ Replaces the rotation gesture recognizer used by the map view and adds it to
+ the UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UIRotationGestureRecognizer* rotationGestureRecognizer;
+
+/**
+ Replaces the shove gesture recognizer used by the map view and adds it to the
+ UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UIPanGestureRecognizer* shoveGestureRecognizer;
+
+/**
+ Replaces the long press gesture recognizer used by the map view and adds it to
+ the UIView. Must not be `nil`.
+ */
+@property (strong, nonatomic) UILongPressGestureRecognizer* longPressGestureRecognizer;
+
+#pragma mark Add Data Layers
+
+/**
+ Adds a named data layer to the map.
+
+ See `TGMapData` for more details.
+
+ @param name The name of the data layer.
+ @return The map data, or nil if the data source can't be initialized.
+
+ @note You cannot create more than one data source with the same name, the same
```

Comment:
Nit: More descriptive note? Maybe like "The previous data source by the same name will be returned"